x-pack/solutions/security/packages/kbn-securitysolution-list-constants/index.ts

```diff
@@ -109,6 +102,18 @@ export const ENDPOINT_ARTIFACT_LIST_IDS = Object.freeze(
   Object.values(ENDPOINT_ARTIFACT_LISTS).map(({ id }) => id)
 );
 
+/**
+ * This ID is used for _both_ the Saved Object ID and for the list_id
+ * for the single global space agnostic endpoint list
+ */
+export const ENDPOINT_LIST_ID = ENDPOINT_ARTIFACT_LISTS.endpointExceptions.id;
+
+/** The name of the single global space agnostic endpoint list */
+export const ENDPOINT_LIST_NAME = ENDPOINT_ARTIFACT_LISTS.endpointExceptions.name;
+
+/** The description of the single global space agnostic endpoint list */
+export const ENDPOINT_LIST_DESCRIPTION = ENDPOINT_ARTIFACT_LISTS.endpointExceptions.description;
+
```

Comment:
All the values below are deprecated, and we’ve been gradually replacing their occurrences with the new constants - e.g., `ENDPOINT_ARTIFACT_LISTS.endpointExceptions.id` instead of `ENDPOINT_LIST_ID`.